json-csharp/EnsuringStringAsValidJson/JsonValidators/JsonSchemaSimpleValidationUseCase.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Text.Json;
+using System.Text.Json.Nodes;
+using Json.Schema;
+using JsonValidators.Abstracts;
+
+namespace JsonValidators;
+
+public class JsonSchemaSimpleValidationUseCase : IJsonValidator
```

Comment:
I apologize, but I believe this class pertains to the use case of validating a JSON string against a defined schema rather than being related to schema builder.

Reply:
Hi @PooyaAlamirpour, while I see the point, I was trying to make it so that the names of the classes are clearer, since we are using Newtonsoft.Json.Schema and the Json.Schema library in the project, there might be some confusion between the class named "JsonSchemaAgainstDesired" and "JsonSchemaSimple". The former is using Newtonsoft, the latter using Json.Schema. By changing the name of this class we make it clear that we are not referring to Newtonsoft. We could rename the other class then, whichever way you think is best, but either way we should change the name of something to help avoid confusion. I would also be okay with renaming both of them, a suggestion would be:
NewtonsoftJsonSchemaUseCase, and JsonSchemaUseCase